src/app/dto/user.dto.go

```diff
@@ -1,14 +1,19 @@
 package dto
 
-type UserDto struct {
+type User struct {
+	Id        string `json:"id"`
+	Email     string `json:"email"`
+	Firstname string `json:"firstname"`
+	Lastname  string `json:"lastname"`
+}
+
+type UpdateUserRequest struct {
 	Email     string `json:"email" validate:"required,email"`
 	Password  string `json:"password" validate:"required,gte=6,lte=30"`
 	Firstname string `json:"firstname" validate:"required"`
 	Lastname  string `json:"lastname" validate:"required"`
 }
 
-type UpdateUserDto struct {
-	Password  string `json:"password" validate:"required,gte=6,lte=30"`
-	Firstname string `json:"firstname" validate:"required"`
-	Lastname  string `json:"lastname" validate:"required"`
+type DeleteUserResponse struct {
+	Success bool `json:"success" validate:"required"`
```

Comment:
I think we do not need validate in the response

Reply:
yeah you're right. Thank you!